### ISSUE TITLE
variables from mixins not recognized in component template

### ISSUE THREAD

**Issue by dpatwp** [vuejs/vetur#1232]
## Info

- Platform: Linux 
- Vetur version: 0.19.0 
- VS Code version: 1.33.1

## Problem

Since the last release of Vetur html template no longer uses values from mixins

example error in template where mixin has `data.config`
```Property 'config' does not exist on type 'CombinedVueInstance...'.```

## Reproducible Case

- create a `base` component with a data and/or computed variable
- create a `extends` component that uses the base `component` as a mixin
- use variable/computed in template of `extends` component

**Comment by octref** [comment]
That's correct, there's no support for mixin yet. It's not easy to do this...

**Comment by dasDaniel** [comment]
I guess roll-back to 0.18.1 for now if needed

**Comment by octref** [comment]
You can also do `vetur.validation.template: false` or vetur.experimental.templateInterpolationService: false`

**Comment by ktsn** [comment]
FYI: If you want to type mixins, you may want to use this. https://github.com/ktsn/vue-typed-mixins

**Comment by ClausClaus** [comment]
I am using mac. After I upgraded to the latest vscode, I also encountered this problem. Now, whether I downgrade to 1.3.1 or use vetur0.18.1, the properties of mixins are always unrecognizable. . .

**Comment by octref** [comment]
Related to #1156

**Comment by boneus** [comment]
> You can also do `vetur.validation.template: false` or `vetur.experimental.templateInterpolationService: false`

Hm, I tried that and I still get errors regarding a property not being on type.

Also, where can I find all the settings Vetur supports?
